torch/_dynamo/variables/nn_module.py

```diff
@@ -983,3 +985,111 @@ def __setattr__(self, name: str, value: Any) -> None:
             value = FSDPManagedNNModuleVariable._wrap_source(value)
 
         return super().__setattr__(name, value)
+
+    def call_method(
+        self, tx, name, args: List[VariableTracker], kwargs: Dict[str, VariableTracker]
+    ) -> VariableTracker:
+        key = self.module_key
+
+        named_embed = functools.partial(
+            _named_embed,
+            tx=tx,
+            key=key,
+            source_cls=FSDPNNModuleSource,
```

Comment:
we don't need all these, but need a different way to propagate `FSDPNNModuleSource`